src/main/java/com/epam/test_generator/services/TestResultService.java

```diff
@@ -0,0 +1,103 @@
+package com.epam.test_generator.services;
+
+import com.epam.test_generator.config.security.AuthenticatedUser;
+import com.epam.test_generator.dao.interfaces.TestResultDAO;
+import com.epam.test_generator.dto.RawSuitResultDTO;
+import com.epam.test_generator.dto.TestResultDTO;
+import com.epam.test_generator.entities.Project;
+import com.epam.test_generator.entities.TestResult;
+import com.epam.test_generator.entities.factory.TestResultFactory;
+import com.epam.test_generator.services.exceptions.BadRequestException;
+import com.epam.test_generator.transformers.TestResultTransformer;
+import java.util.Date;
+import java.util.List;
+import java.util.stream.Collectors;
+import javax.transaction.Transactional;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.Authentication;
+import org.springframework.stereotype.Service;
+
+@Transactional
+@Service
+public class TestResultService {
+
+    @Autowired
+    private TestResultFactory testResultFactory;
+
+    @Autowired
+    private TestResultDAO testResultDAO;
+
+    @Autowired
+    private TestResultTransformer testResultTransformer;
+
+    /**
+     * Get all test results by projectId.
+     *
+     * @param projectId identifier of {@link Project}
+     * @return list of {@link TestResult}
+     */
+    public List<TestResultDTO> getTestResults(long projectId) {
```

Comment:
Why we have 3 methods here for getting projects? Isn't it sufficient to have just _getTestResults(long projectId)_ ?